.editorconfig

```diff
@@ -1,7 +0,0 @@
-[*]
-end_of_line = lf
-charset = utf-8
-trim_trailing_whitespace = true
-insert_final_newline = true
-indent_style = space
-indent_size = 4
```

Comment:
I don't think you meant to delete this

Reply:
You're right, I had to reset my eclipse project, deleted that in my overzealousness in resetting the project base. Fixed it :-)